content/en/docs/concepts/containers/images.md

```diff
@@ -61,9 +61,9 @@ you can do one of the following:
 
 When `imagePullPolicy` is defined without a specific value, it is also set to `Always`.
 
-## Multi-architecture Images with Manifests
+## Multi-architecture Images with Image Indexes
```

Comment:
Hi @nate-double-u ,
Would you check that the headings use sentence case? Thanks!

Reply:
Titles updated throughout.